omisego-sdk/src/main/java/co/omisego/omisego/websocket/listener/SocketCustomEventListener.kt

```diff
@@ -7,16 +7,12 @@ package co.omisego.omisego.websocket.listener
  * Copyright © 2017-2018 OmiseGO. All rights reserved.
  */
 
-import co.omisego.omisego.model.APIError
 import co.omisego.omisego.model.socket.SocketReceive
-import co.omisego.omisego.model.transaction.consumption.TransactionConsumption
 import co.omisego.omisego.websocket.event.SocketEvent
-import co.omisego.omisego.websocket.event.TransactionConsumptionFinalizedEvent
-import co.omisego.omisego.websocket.event.TransactionConsumptionRequestEvent
 import co.omisego.omisego.websocket.strategy.FilterStrategy
 
 interface SocketCustomEventListener {
-    var strategy: FilterStrategy
+    val strategy: FilterStrategy
```

Comment:
Still to move to `SimpleSocketCustomEventListener`

Reply:
Moved that